yellowbrick/features/rankd.py

```diff
@@ -205,12 +240,12 @@ def transform(self, X, **kwargs):
 
         Returns
         -------
-        Xp : ndarray
+        X' : ndarray
             The transformed matrix, X'
         """
         # Rank and draw the input matrix
-        Xp = self.rank(X)
-        self.draw(Xp, **kwargs)
+        R = self.rank(X)
+        self.draw(R, **kwargs)
```

Comment:
So why `R` here, for rank matrix? I get that `X'` stands for X prime, as does presumably `Xp`. My first reaction was that this is a bit of a faux pas, simply changing a variable name to something you prefer is bad practice with open source as it can lead to confusion or worse, start a turf fight. However, if the original names were leading to confusion, then it is a good thing to introduce clarity, and in this case, I think some clarity is needed, and should be enhanced with additional documentation. In that spirit I suggest the following:

```python
def transform(self, X, **kwargs):
    """
    Calling the ``transform()`` method is the primary hook for drawing feature rankings. 

    ...

    Returns 
    -------
    X : ndarray 
        Typically a transformed matrix, X' is returned. However, this method
        performs no transformation on the original data, instead simply 
        ranking the features that are in the input data and returns the original 
        data, unmodified. 
    """
    
    # R is a n dimensional, symmetric array where n is the number of features. 
    # E.g. for 1D ranking, R is (n,) and for 2D ranking R is (n,n) and so forth. 
    R = self.rank(X)
    self.draw(R, **kwargs)
    ... 
``` 

What do you think? Does that help make things more obvious to both users and developers?